.rubocop.yml

```diff
@@ -17,5 +17,15 @@ Style/AlignParameters:
 Style/MultilineOperationIndentation:
   EnforcedStyle: indented
 
+Style/RedundantReturn:
+  Exclude:
+    - "chef/data_bags/crowbar/migrate/*"
+
 Lint/EndAlignment:
   AlignWith: variable
+
+Lint/UnusedMethodArgument:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 100
```

Comment:
I'm not a real fan of this as well. I would like to avoid that long lines. To make the transition a little bit smoother we can use an approach like we are using at hawk (https://github.com/ClusterLabs/hawk/blob/master/.rubocop.yml#L1). We added a .rubocop_todo.yml for style fixes that will be delayed for later.